projects/exui-common-lib/src/lib/components/write-address/write-address.component.scss

```diff
@@ -2,7 +2,10 @@
   cursor: pointer;
   display: block;
   text-decoration: underline;
-  margin-top: 10px;
+}
+
+.postcodeLookup {
+  margin-bottom: -30px;
```

Comment:
@connorpgpmcelroy  Can we have an alternative to this? Rather than using negative values for margin?

Reply:
_The downside of negative margins is they are difficult to debug and will make your CSS harder to read._